File: laptop/C_Solution/단계별/기본수학1/2292.c
// Bee Home

#include <stdio.h>
#include <stdlib.h>

int main(){
    int n, cnt=1, x=6;
    scanf("%d", &n);

    if(n == 1){
        printf("%d\n", 1);
        exit(0);
    }
    while(1){
        cnt += x;
        if(cnt >= n){
            printf("%d\n", x/6+1);
            break;
        }
        x += 6;
    }
    return 0;
}

File: laptop/Python_Solution/단계별/기본수학1/2292.py
# Bee Home

cnt=1
x=6
n=int(input())
while(1):
    cnt += x
    if(cnt >= n):
        print(x//6+1 if n!=1 else 1)
        break
    x+=6
    
#a=int(input());print(1 if a==1 else[i for i in range(18259)if 3*i*(i+1)+1>=a][0]+1)